rust-sdk/tx-sender/src/signer.rs

```diff
@@ -0,0 +1,31 @@
+use solana_sdk::{
```

Comment:
note: i ended up not needing this after the change to the build/send function signatures. feel free to delete if you like.